queue/abstract.lua

```diff
@@ -511,7 +511,20 @@ function method.start()
         })
     end
 
-    _queue:pairs():each(recreate_tube)
+    for _, tube_tuple in _queue:pairs() do
+        local space_name = tube_tuple[3]
+        tube = recreate_tube(tube_tuple)
+        local space = box.space[space_name]
+        for _, task_tuple in space:pairs() do
+            -- Release all taken tasks on start
+            -- See https://github.com/tarantool/queue/issues/66
+            -- for more information
+            local tid, task_state = task_tuple[1], task_tuple[2]
+            if task_state == state.TAKEN then
+                tube.raw:release(tid)
```

Comment:
Sooner or later we'll return to #85 and I guess releasing tasks after restart will need to be optional after that. But not now, just noted.